### ISSUE TITLE
Emissive shaders broken

### ISSUE THREAD

**Issue by ddans151** [GTAmodding/GTAIV-Issues-List#4]
gta_emissivestrong shader is broken after Patch 6 (e.g. no Rotterdam Tower lights)
![GTAIV 2019-08-13 22-30-36-99](https://user-images.githubusercontent.com/54078953/62971445-933a2800-be1a-11e9-8078-b303aaf4efd3.png)

You can use the shader from old patches, but then traffic lights are broken
![GTAIV 2019-08-13 22-33-15-58](https://user-images.githubusercontent.com/54078953/62971533-c086d600-be1a-11e9-83e3-7a0bdec1748f.png)

Also this one, the lights behind BAWSAQ is gta_emissive
![GTAIV 2019-08-15 14-54-58-44](https://user-images.githubusercontent.com/54078953/63093313-503c9980-bf6d-11e9-98d0-0abdee07707f.png)

Basically there's something wrong with the way emissive things render on new patches.

**Comment by Tomasak** [comment]
Well this fixes it by using old shader and applying different shader to traffic lights
https://gtaforums.com/topic/569019-betaiveflc-graphical-bug-fixes/

Not sure if this is correct way of fixing this.

**Comment by ddans151** [comment]
It's always possible there's more models broken with emissive, so a proper fix would be better instead of nkjellman's workaround.

**Comment by ThirteenAG** [comment]
![](https://media.discordapp.net/attachments/360065330371887107/612444563055050772/broken.jpg)

![](https://cdn.discordapp.com/attachments/360065330371887107/612444828143190076/fixed.jpg)

**Comment by ThirteenAG** [comment]
Xbox one screens
![8-18-2019_10-54-03_PM-uwqvp0gx](https://user-images.githubusercontent.com/4904157/63229649-77b09200-c20b-11e9-8351-115dc6c8612b.png)
![8-18-2019_10-55-06_PM-bdymf0sv](https://user-images.githubusercontent.com/4904157/63229667-a3cc1300-c20b-11e9-9e7d-18be2dcdfe07.png)
![8-18-2019_10-59-07_PM-jwpd0fmm](https://user-images.githubusercontent.com/4904157/63229732-41274700-c20c-11e9-9639-2b54bf1161d8.png)

**Comment by donnits** [comment]
bawsaq emissve fix: http://www.mediafire.com/file/56s0nyhc78ez9y3/bawsaq_emissive.rar/file

**Comment by philou-felin** [comment]
I just want to report that Jizzy Jims is still broken in the current version of the game available on Steam. I assume a lot of similar bugs are still present too. (I did not try to mod my game yet.)
